src/tdastro/astro_utils/passbands.py

```diff
@@ -48,26 +51,37 @@ def __init__(
             - units : str (either 'nm' or 'A')
             If survey is not LSST (or other survey with defined defaults), either a table_path or table_url
             must be provided.
+        table_dir : str, optional
+            The path to the directory containing the passband tables. If a table_path has not been specified
+            in the passband_parameters dictionary, table paths will be set to
+            {table_dir}/{survey}/{filter_name}.dat. If None, the table path will be set to a default path.
```

Comment:
No change needed right now, but I had been thinking of this as "{table_dir}/{filter_name}.dat" so the user could effectively provide both the base directory and the survey name with the table_dir parameter.

This current approach better matches the presets below though.